package.json

```diff
@@ -150,7 +151,8 @@
   },
   "babel": {
     "plugins": [
-      "lodash", [
```

Comment:
`package.json` should not ever be committed unless for some reason you're adding core functionality that adds a new npm package.

Reply:
Same here! :D